### ISSUE TITLE
Refine labels with modelzoo

### ISSUE THREAD

**Issue by Magamor** [DeepLabCut/DeepLabCut#2806]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I used the model full_macaque and now i tried to refine the labels by following the instructions here : https://deeplabcut.github.io/DeepLabCut/docs/recipes/UsingModelZooPupil.html 

 I added the path of the videos to config.yaml but  when i try to extract outlier frames i have this message : 
```
No unfiltered data file found in C:\Users\magali.moreau\Desktop\test-mag-2024-12-05\videos for video Anyanka_Run010_body1 and scorer DLC_resnet50_Anyanka_bodyNov20shuffle1_1030000.
It seems the video has not been analyzed yet, or the video is not found! You can only refine the labels after the a video is analyzed. Please run 'analyze_video' first. Or, please double check your video file path. 
```

It seems that it's trying to find files that are not created when i use a pretrained model, because the analysis is supposed to be done.

Thank you for your help.

### Operating System

operating system

### DeepLabCut version

dlc version

### DeepLabCut mode

single animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @Magamor - could you please let me know which version of DeepLabCut you're using, and which operating system you're on?

**Comment by Magamor** [comment]
Ah yes sorry, i am using DLC 3.0.0rc6 on Windows. I installed a driver (Nvidia driver 553.35) and CUDA 12.4 my GPU is Quadro P2000

**Comment by n-poulsen** [comment]
Thanks - did you train a model with the TensorFlow engine or with the PyTorch engine?

And which files do you have in the `C:\Users\magali.moreau\Desktop\test-mag-2024-12-05\videos` directory?

**Comment by MHRosenberg** [comment]
@n-poulsen, I believe I may have a related issue: [here](https://github.com/DeepLabCut/DeepLabCut/issues/2796#issuecomment-2570590314)

**Comment by maximpavliv** [comment]
A similar case (mentioned by @MHRosenberg ) has been solved with [#2826](https://github.com/DeepLabCut/DeepLabCut/pull/2826) , does it also solve your issue @Magamor ? (please update your DLC installation with `pip install --upgrade "git+https://github.com/DeepLabCut/DeepLabCut.git@pytorch_dlc#egg=deeplabcut"` and try again)

**Comment by n-poulsen** [comment]
I'll close this issue as I believe it's been fixed in #2826, as mentioned by @maximpavliv - please re-open it if you still have this issue.

**Comment by MHRosenberg** [comment]
Not fixed as far as I can tell. https://github.com/DeepLabCut/DeepLabCut/issues/2796#issuecomment-2593703565
